### PR TITLE
Added doctestable examples to mapping in operation.

### PR DESCRIPTION
**Pull request by mHash1m**
Ready to merge

### CODE REVIEW

File: dffml/operation/mapping.py [hunk 1]
```diff
@@ -25,4 +69,45 @@ def mapping_extract_value(mapping: Dict[str, Any], traverse: List[str]):
...
+    ++++++++
+
+    >>> dataflow = DataFlow.auto(create_mapping, GetSingle)
+    >>> dataflow.seed.append(
+    ...     Input(
+    ...         value=[create_mapping.op.outputs["mapping"].name],
+    ...         definition=GetSingle.op.inputs["spec"],
+    ...     )
+    ... )
+    >>> Inputs = [
```
Comment (pdxjohnny) [inline dffml/operation/mapping.py:97]:
```suggestion
    >>> inputs = [
```

File: dffml/operation/mapping.py [hunk 2]
```diff
@@ -25,4 +69,45 @@ def mapping_extract_value(mapping: Dict[str, Any], traverse: List[str]):
...
+    ...     Input(
+    ...         value="key1", definition=create_mapping.op.inputs["key"], parents=None,
+    ...     ),
+    ...     Input(
+    ...         value=42, definition=create_mapping.op.inputs["value"], parents=None,
+    ...     ),
+    ... ]
+    >>>
+    >>> async def main():
+    ...     async for ctx, result in MemoryOrchestrator.run(dataflow, Inputs):
```
Comment (pdxjohnny) [inline dffml/operation/mapping.py:107]:
```suggestion
    ...     async for ctx, result in MemoryOrchestrator.run(dataflow, inputs):
```

File: dffml/operation/mapping.py (hunk 1 above)
Comment (pdxjohnny) [inline dffml/operation/mapping.py:97]:
```suggestion
    >>> inputs = [
```

File: dffml/operation/mapping.py [hunk 3]
```diff
@@ -16,6 +16,50 @@
...
+    ...         definition=mapping_extract_value.op.inputs["mapping"],
+    ...     ),
+    ...     Input(
+    ...         value=["key1", "key2"],
+    ...         definition=mapping_extract_value.op.inputs["traverse"],
+    ...     ),
+    ... ]
+    >>>
+    >>> async def main():
+    ...     async for ctx, result in MemoryOrchestrator.run(dataflow, Inputs):
```
Comment (pdxjohnny) [inline dffml/operation/mapping.py:57]:
```suggestion
    ...     async for ctx, result in MemoryOrchestrator.run(dataflow, inputs):
```

File: dffml/operation/mapping.py [hunk 4]
```diff
@@ -16,6 +16,50 @@
...
+
+    >>> dataflow = DataFlow.auto(mapping_extract_value, GetSingle)
+    >>>
+    >>> dataflow.seed.append(
+    ...     Input(
+    ...         value=[mapping_extract_value.op.outputs["value"].name],
+    ...         definition=GetSingle.op.inputs["spec"],
+    ...     )
+    ... )
+    >>> Inputs = [
```
Comment (pdxjohnny) [inline dffml/operation/mapping.py:45]: Minor thing, we never camelcase variables

```suggestion
    >>> inputs = [
```

**Comment by mHash1m** [comment]
@pdxjohnny can you please look at the error, seems like it has to do something with tfdnnc test. Not sure if I should be tweaking there for this.

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Added
+- Doctestable examples to `operation/mapping.py`.
```
Comment (pdxjohnny) [inline CHANGELOG.md:9]:
```suggestion
- Doctestable examples to `operation/mapping.py`.

```

**Pull request merged**